run_evals_from_checkpoints.py

```diff
@@ -0,0 +1,380 @@
+    parallel: bool,
+):
+    """Generate evaluation configs for each checkpoint."""
+    for folder in checkpoints.glob("*"):
+        if folder.is_dir():
+            cmd = [
+                "python",
+                "generate_eval_config_from_checkpoint.py",
+                "--checkpoint",
+                folder,
+                "--output-dir",
+                str(checkpoints),
+            ]
+
+            # Add optional arguments if they're provided
+            if train_config:
+                cmd.extend(["--train-config", str(train_config)])
+            if wandb_run:
+                cmd.extend(["--wandb-run", wandb_run])
+            if wandb_project:
+                cmd.extend(["--wandb-project", wandb_project])
+            if wandb_entity:
+                cmd.extend(["--wandb-entity", wandb_entity])
+            if track_run:
+                cmd.append("--track-run")
+
+            # classification head options
+            if pooling_type:
+                cmd.extend(["--pooling-type", pooling_type])
+            if head_class_act:
+                cmd.extend(["--head-class-act", head_class_act])
+            if head_class_norm:
+                cmd.extend(["--head-class-norm", head_class_norm])
+            if head_class_dropout > 0:
+                cmd.extend(["--head-class-dropout", str(head_class_dropout)])
+
+            # Add skip arguments
+            if skip_semipro:
+                cmd.append("--skip-semipro")
+            if skip_reserve:
+                cmd.append("--skip-reserve")
+            if skip_eurlex:
+                cmd.append("--skip-eurlex")
+            if skip_mnli:
+                cmd.append("--skip-mnli")
+            if skip_boolq:
+                cmd.append("--skip-boolq")
+            if skip_wic:
+                cmd.append("--skip-wic")
+
+            for seed in seeds:
+                cmd.extend(["--seeds", str(seed)])
+
+            cmd.append("--parallel") if parallel else cmd.append("--single")
+
+            # Run the config generation process without suppressing output
+            run_subprocess(cmd)
+
+
+# fmt: off
+@app.command()
+def main(
+    checkpoints: Annotated[Path, Option(help="Path to the directory containing checkpoints", rich_help_panel="Checkpoint & Config Paths")],
+    train_config: Annotated[Optional[Path], Option(help="Path to a single .yaml file containing training configuration", rich_help_panel="Checkpoint & Config Paths")] = None,
+    skip_generation: Annotated[bool, Option("--skip-generation", help="Skip generation of evaluation configs", rich_help_panel="Config Options")] = False,
+    wandb_run: Annotated[Optional[str], Option(help="wandb run containing the training configuration", rich_help_panel="W&B")] = None,
+    wandb_project: Annotated[Optional[str], Option(help="wandb project for the run", rich_help_panel="W&B")] = None,
+    wandb_entity: Annotated[Optional[str], Option(help="wandb entity for the project", rich_help_panel="W&B")] = None,
```

Comment:
nit: would we not want this to default to the main project name as a slight time saver?

Reply:
it's one less thing to un-hardcode on release. Plus you can set it in a yaml for `run_evals_from_checkpoints.py` and not have to pass it in every time